src/Tribe/Views/V2/Template_Bootstrap.php

```diff
@@ -69,39 +69,42 @@ public function disable_v1() {
 	}
 
 	/**
-	 * Determines with backwards compatibility in mind, which template user has selected
-	 * on the Events > Settings page as their base Default template
+	 * Determines, with backwards compatibility in mind, which template user has selected
+	 * on the Events > Settings page as their base Default template.
 	 *
 	 * @since  4.9.2
+	 * @since TBD specifically pass 'event' to be clearer down the line where we check for custom templates.
 	 *
-	 * @return string Either 'event' or 'page' based templates
+	 * @return string Either 'event', 'page' or custom based templates
 	 */
 	public function get_template_setting() {
-		$template = 'event';
 		$default_value = 'default';
-		$setting = tribe_get_option( 'tribeEventsTemplate', $default_value );
+		$template = tribe_get_option( 'tribeEventsTemplate', $default_value );
```

Comment:
:warning: Equals sign not aligned with surrounding assignments; expected 6 spaces but found 1 space

<sub>hash: ca8b2028e2f7da25ed9d95fb0f47e713</sub>